here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/util/HandlerUtil.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (C) 2017-2023 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+package com.here.naksha.lib.handlers.util;
+
+import com.here.naksha.lib.core.exceptions.NoCursor;
+import com.here.naksha.lib.core.exceptions.XyzErrorException;
+import com.here.naksha.lib.core.models.XyzError;
+import com.here.naksha.lib.core.models.geojson.implementation.XyzFeature;
+import com.here.naksha.lib.core.models.geojson.implementation.XyzProperties;
+import com.here.naksha.lib.core.models.geojson.implementation.namespaces.EChangeState;
+import com.here.naksha.lib.core.models.geojson.implementation.namespaces.EReviewState;
+import com.here.naksha.lib.core.models.geojson.implementation.namespaces.HereDeltaNs;
+import com.here.naksha.lib.core.models.geojson.implementation.namespaces.XyzNamespace;
+import com.here.naksha.lib.core.models.storage.*;
+import java.util.ArrayList;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class HandlerUtil {
+
+  public static String REVIEW_STATE_PREFIX = "@:review-state:";
+
+  public static @NotNull ContextXyzFeatureResult createContextResultFromCodecList(
+      final @NotNull List<?> inputCodecs,
+      final @Nullable List<?> context,
```

Comment:
`context` param is not used - do we need this?

Reply:
context is required as per design. Not for mock though.